tests/test_snakefmt.py

```diff
@@ -54,14 +54,14 @@ def test_stdinAsSrc_WritesToStdout(self, cli_runner):
         assert actual.output == expected_output
 
     def test_src_dir_arg_files_modified_inplace(self, cli_runner):
-        with tempfile.TemporaryDirectory() as tmpdir:
+        with tempfile.TemporaryDirectory(dir="/tmp") as tmpdir:
```

Comment:
Can you remind me why this is being done?

What if `/tmp` cannot be created on the host machine the tests run on? Can we guarantee all host machines we run tests on will permit this?

Reply:
It's basically trying to create a directory structure to test that we are searching for snakefiles and ignoring things that should be ignored etc.

I used `/tmp` as it seems likely to exist in most places and will be auto cleaned up by the OS usually. I can't use the normal temp dir because as I mentioned above, the Mac CI tests fail.

Happy to change the behaviour to something else though if you have a better idea?

Reply:
everything else looks fine to me so approved